JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -1006,7 +1007,7 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
   if (options$confidenceIntervalsPostHoc || makeBootstrapTable) {
     
     if (makeBootstrapTable) {
-      thisOverTitle <- gettextf("%1$s%% bca%2$s CI", options$confidenceIntervalIntervalPostHoc * 100, "\u002A")
+      thisOverTitle <- gettextf("%1$s%% bca%2$s CI", options$confidenceIntervalIntervalPostHoc * 100, "\u2020")
```

Comment:
perhaps add:

```r
contrastTable$addFootnote(symbol = "\u2020", message = gettext("Bias corrected accelerated."))
`
so people don't need to look/ enable other tables to know what `bca` stands for.

Reply:
That was already in there, but I moved it to that same if statement (there was the same if statement later to added some more stuff based on makeboostraptable).